https/README.md

```diff
@@ -0,0 +1,24 @@
+# HTTPS Package for Prometheus
+
+The `https` directory contains a Go package and a sample configuration file for running `node_exporter` with HTTPS instead of HTTP.
+When running a server with TLS use the flag `--web.config`
+
+e.g. `./node_exporter --web.config="https/tls-config.yml"`
```

Comment:
I'd not have the subdir here, it's likely to confuse users into thinking it's required.